### DISCUSSION TITLE
ToggleColumn  to change all status

### DISCUSSION THREAD

**Discussion by mondherinfo** [filamentphp/filament#5007] [General]
Hello,
ToggleColumn in list page, i want to click ToggleColumn::make('status') to activate an article and deactivate all other, is it possible ?

**Comment by danharrin** [comment]
A model observer is probably best for this?

**Reply by mondherinfo** [reply]
what is the best way to refresh the view after observer finished updating (refresh only view not the page),
thank you

**Reply by danharrin** [reply]
I dont think that toggle columns can currently be refreshed like that yet...

**Comment by mondherinfo** [comment]
hello, thank you, but  afterStateUpdated is not a defined function, I'm
talking about a problem in table builder, not form builder,
togglecolumn(table builder) not toggle(form builder)

Le dim. 20 nov. 2022 à 09:25, Saeed Shamsi ***@***.***> a
écrit :
